suzieq/poller/worker/nodes/node.py

```diff
@@ -1118,10 +1121,8 @@ async def _init_ssh(self, init_boot_time=True, _: bool = True) -> None:
         of retries to enable things like run-once=gather to work.
         '''
         backoff_period = 1
-        while True:
+        while not self._conn:
             await super()._init_ssh(init_boot_time=False, rel_lock=False)
-            if self._conn:
-                break
 
             await asyncio.sleep(backoff_period)
             backoff_period *= 2
```

Comment:
Removing the `break` we are always waiting the backoff_period. Is it what we want? If yes, why?

Reply:
Mistake. FIxed